BREAKING.md

```diff
@@ -1,10 +1,25 @@
 ## 0.43 Breaking changes
 
 - [TinyliciousClient no longer static](#TinyliciousClient-no-longer-static)
+- [Property removed from the Container class](#Property-removed-from-the-Container-class)
+- [The legacy way of creating containers has been deprecated](#The-legacy-way-of-creating-containers-has-been-deprecated)
+- [The legacy way of creating documents has been deprecated](#The-legacy-way-of-creating-documents-has-been-deprecated)
 
 ### TinyliciousClient no longer static
 `TinyliciousClient` global static property is removed. Instead, object instantiation is now required.
 
+### Property removed from the Container class
+- the `existing` property from `Container` has been removed.
```

Comment:
Can you please add some remedy to consider? I.e. caller should differentiate based on how it creates container - through load() or createDetached() API. deprecated (but for now supported) escape valve (see next point) does not provide a way to differentiate, developers are advised to move away from this flow.